### DISCUSSION TITLE
Is tootctl media remove safe?

### DISCUSSION THREAD

**Discussion by darkphoenix** [mastodon/mastodon#18059] [Q&A]
My instance is growing a rather huge remote media cache, however I'm not entirely sure what the consequences of pruning it are and the documentation doesn't exactly say anything about that.  If I erase old media like this, will it automatically be re-fetched whenever it is needed again, or is this a destructive operation on the assumption old posts are irrelevant?

**Comment by Gargron** [comment]
Yes, it will be refetched when needed (assuming the other server is still alive). It's considered safe to run.
